docs/src/main/sphinx/connector/metastores.md

```diff
@@ -467,3 +464,12 @@ using Hive 3.x:
      <value>org.apache.hadoop.hive.metastore.SerDeStorageSchemaReader</value>
  </property>
 ```
+
+## Limitations
+
+- Glue does not support setting comments on existing tables. To use the COMMENT
```

Comment:
I would remove the - in front of this line. 

Also we really need the example?

Overall we could just do

Glue does not support setting comments on existing tables. You can only set comments when creating a table.

Or 

Glue does not support setting comments on existing tables. You can only set comments when creating a table: 

with the code snippet ..

it would would have to be formatted using sql. and broken over a few lines. Or we could check and see if an example is in the CREATE TABLE docs itself .. add it there and link to that page.